state/session/reducer.js

```diff
@@ -1,26 +1,24 @@
+import cookie from 'cookie_js';
 import * as types from './types';
 
+const checkSessionIdInCookie = cookie.get('session_id');
+
 const initialState = {
-  sessionId: '',
+  sessionId: checkSessionIdInCookie,
   requestToken: null
 };
 
 function sessionReducer(state = initialState, action) {
   switch (action.type) {
-    case types.SESSION_REQUEST_TOKEN:
+    case types.REQUEST_SESSION:
       return {
         ...state,
         requestToken: action.text
```

Comment:
You don't need to store this value because this action calls logic and passes values to it

Reply:
Done